src/Exception/MethodNotAllowedHttpException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Akeneo\Pim\ApiClient\Exception;
+
+/**
+ * @copyright 2022 Akeneo SAS (https://www.akeneo.com)
+ * @license   https://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class MethodNotAllowedHttpException extends ClientErrorHttpException
+{
+    protected function getAdditionalInformationMessage(): string
+    {
+        return '(see https://api.akeneo.com/php-client/exception.html#method-not-allowed-exception)';
```

Comment:
is it normal that the links does not target a correct anchor? it's probably in a branch?

Reply:
Yes the doc is going to evolve, there's no branch yet but it will be there with the next client release.
Though I've been wondering: It's very likely the docs gets deployed before the new client version gets released, or the opposite. Is there any way to synchronise both?